src/terminal.c

```diff
@@ -526,7 +525,9 @@ main (int argc, char **argv)
 
 	if (options->startup_id == NULL)
 	{
-		options->startup_id = g_strdup_printf ("_TIME%lu", g_get_monotonic_time () / 1000);
+		options->startup_id
+			= g_strdup_printf ("_TIME%" G_GINT64_FORMAT,
```

Comment:
Do we really need an extra line for `= g_strdup_printf........`?

Reply:
done